modules/microshift-oc-mirror-list-ops-catalogs.adoc

```diff
@@ -0,0 +1,47 @@
+//Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-oc-mirror-list-operators-catalogs_{context}"]
+= Using oc mirror to select Operators
+
+Use the following example procedure to select a catalog and Operators to add to your oc mirror image set configuration file.
+
+.Prerequisites
+* The {oc-first} is installed.
+* Operator Lifecycle Manager (OLM) is installed.
+* The oc-mirror OpenShift CLI (oc) plugin is installed.
+
+.Procedure
+. Get a list of available catalogs to filter by running the following command:
++
+[source,terminal]
+----
+$ oc mirror list operators --version 4.15 --catalogs
+----
+//Q: can we use the --version 4.15 --catalogs flags for homemade Operators? Do we need to have loaded them onto oc mirror in a previous step?
```

Comment:
`list operators --version 4.15 --catalogs` returns list of "official" OpenShift catalogs. When user has homemade catalog, they don't need OpenShift catalogs